modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -103,9 +104,10 @@ use \{{invokerPackage}}\ObjectSerializer;
      * Operation {{{operationId}}}WithHttpInfo
      *
      * {{{summary}}}.
-     *
-  {{#allParams}}   * @param {{dataType}} ${{paramName}} {{description}} {{#required}}(required){{/required}}{{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
-  {{/allParams}}   *
+     */
+  {{#allParams}} //  * @param {{dataType}} ${{paramName}} {{description}} {{#required}}(required){{/required}}{{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
+  {{/allParams}}   
+    /**
```

Comment:
@sdavis-r7 I'll probably keep `*` and sanitize the tag to remove line break instead so as to conform to the PHP coding style.